web/tests/functional/cppcheck/__init__.py

```diff
@@ -0,0 +1,79 @@
+# coding=utf-8
+# -----------------------------------------------------------------------------
+#                     The CodeChecker Infrastructure
+#   This file is distributed under the University of Illinois Open Source
+#   License. See LICENSE.TXT for details.
+# -----------------------------------------------------------------------------
+
+"""Setup for the package tests."""
+from __future__ import print_function
+from __future__ import division
+from __future__ import absolute_import
+
+import multiprocessing
+import os
+import shutil
+
+from libtest import codechecker
+from libtest import env
+
+# Stopping event for CodeChecker server.
+__STOP_SERVER = multiprocessing.Event()
+
+# Test workspace initialized at setup for cppcheck tests.
+TEST_WORKSPACE = None
+
+
+def setup_package():
+    """Setup the environment for the tests then start the server."""
+
+    global TEST_WORKSPACE
+    TEST_WORKSPACE = env.get_workspace('cppcheck')
+
+    os.environ['TEST_WORKSPACE'] = TEST_WORKSPACE
+
+    test_config = {}
+
+    # Setup environment variables for the test cases.
+    host_port_cfg = {'viewer_host': 'localhost',
+                     'viewer_port': env.get_free_port(),
+                     'viewer_product': 'cppcheck'}
+
+    test_env = env.test_env(TEST_WORKSPACE)
+
+    codechecker_cfg = {
+        'check_env': test_env,
+        'workspace': TEST_WORKSPACE,
+        'checkers': []
+    }
+
+    codechecker_cfg.update(host_port_cfg)
+
+    codechecker_cfg['run_names'] = []
+
+    test_config['codechecker_cfg'] = codechecker_cfg
+
+    # Export configuration for the tests.
+    env.export_test_cfg(TEST_WORKSPACE, test_config)
+
+    print("Starting server to get results")
+    codechecker.start_server(codechecker_cfg, __STOP_SERVER)
```

Comment:
Can not we just use the already running global server instance like here?
https://github.com/Ericsson/codechecker/blob/ed2a146b28fdc2f5a39af976139ec32b8b7c9e25/web/tests/functional/comment/__init__.py#L62